db/sg_replicate_cfg.go

```diff
@@ -107,6 +109,7 @@ type ReplicationConfig struct {
 	Cancel                 bool                      `json:"cancel,omitempty"`
 	Adhoc                  bool                      `json:"adhoc,omitempty"`
 	BatchSize              int                       `json:"batch_size,omitempty"`
+	RunAs                  string                    `json:"run_as,omitempty"`
```

Comment:
Review whether it still makes sense to deprecate `username` and move to `remote_username` given this is now called `run_as`.